spec/functional/templating_spec.rb

```diff
@@ -59,7 +59,9 @@ class RandomClass
           expect(destination).to be_an_executable
         end
       end
+    end
 
+    describe "#render_template_content" do
```

Comment:
It looks like this test is calling `render_template` (on line 80) instead of `render_template_content`.